### PR TITLE
Rename Storefront tags

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
Rename Storefront tags so Core API and Storefront operations with same name are not confused with each other.

Combined operations with Account, Authentication, Email verification, Password reset tags into `Storefront account` tag

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -519,6 +503,68 @@ tags:
     description: |-
       Use this operation to check the status of the Rebilly API.
       No authentication is required.
+  - name: Storefront account
+    description: |-
+      Use these operations to manage Rebilly Storefront accounts.
```
Comment (ashkarpetin) [inline openapi/openapi.yaml:508]:
```suggestion
      Use these operations to manage Storefront accounts.
```

File: openapi/openapi.yaml [hunk 2]
```diff
@@ -519,6 +503,68 @@ tags:
...
       Use this operation to check the status of the Rebilly API.
       No authentication is required.
+  - name: Storefront account
+    description: |-
+      Use these operations to manage Storefront accounts.
+      An account in which you are subscribed to in order to use the Rebilly product.
+      APIs intended to be used directly from an untrusted browser.
+  - name: Storefront billing portals
+    description: |-
+      Use these operations to access Storefront billing portals.
```
Comment (ashkarpetin) [inline openapi/openapi.yaml:513]: Not sure if these are good descriptions.

**Review by i-yasko — DISMISSED** [review]

File: openapi/openapi.yaml [hunk 3]
```diff
@@ -519,6 +503,68 @@ tags:
     description: |-
       Use this operation to check the status of the Rebilly API.
       No authentication is required.
+  - name: Storefront account
+    description: |-
+      Use these operations to manage Storefront accounts.
+      An account in which you are subscribed to in order to use the Rebilly product.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:510]: suggestion: to add more detail
```suggestion
      Use these operations to manage storefront accounts.
      A storefront account is an account that the customer is subscribed to in order to use the Rebilly product.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).

      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 4]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront billing portals
+    description: |-
+      Use these operations to access Storefront billing portals.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:514]: suggestion: to add more detail
```suggestion
      Use these operations to access storefront billing portals.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 5]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront checkout forms
+    description: |-
+      Use these operations to access Storefront checkout forms.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:518]: suggestion: to add more detail
```suggestion
      Use these operations to access storefront checkout forms.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 6]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront custom fields
+    description: |-
+      Use these operations to access Storefront custom fields.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:522]:
```suggestion
      Use these operations to access storefront custom fields.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 7]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront deposits
+    description: |-
+      Use these operations to access Storefront deposits.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:526]:
```suggestion
      Use these operations to access Storefront deposits.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 8]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront invoices
+    description: |-
+      Use these operations to access Storefront invoices.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:530]:
```suggestion
      Use these operations to access storefront invoices.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 9]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront KYC documents
+    description: |-
+      Use these operations to access Storefront KYC documents.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:534]:
```suggestion
      Use these operations to access storefront KYC documents.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 10]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront orders
+    description: |-
+      Use these operations to access Storefront orders.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:538]:
```suggestion
      Use these operations to access storefront orders.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 11]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront payment instruments
+    description: |-
+      Use these operations to access Storefront payment instruments.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:542]:
```suggestion
      Use these operations to access storefront payment instruments.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 12]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront plans
+    description: |-
+      Use these operations to access Storefront plans.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:546]:
```suggestion
      Use these operations to access storefront plans.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, 
      the customer must be issued a JWT with limited permissions. 
      For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 13]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront products
+    description: |-
+      Use these operations to access Storefront products.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:550]:
```suggestion
      Use these operations to access storefront products.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, 
      the customer must be issued a JWT with limited permissions. 
      For more information, 
      see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 14]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront purchases
+    description: |-
+      Use these operations to manage Storefront purchases.
+      Purchases are transactions that have been executed related to the purchase of goods or services.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:555]:
```suggestion
      Use these operations to manage storefront purchases.
      Purchases are transactions that have been executed related to the purchase of goods or services.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, 
      the customer must be issued a JWT with limited permissions. 
      For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 15]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront quotes
+    description: |-
+      Use these operations to access Storefront quotes.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:559]:
```suggestion
      Use these operations to access storefront quotes.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, the customer must be issued a JWT with limited permissions. For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 16]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront transactions
+    description: |-
+      Use these operations to access Storefront transactions.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:563]:
```suggestion
      Use these operations to access Storefront transactions.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, 
      the customer must be issued a JWT with limited permissions. 
      For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

File: openapi/openapi.yaml [hunk 17]
```diff
@@ -519,6 +503,68 @@ tags:
...
+  - name: Storefront websites
+    description: |-
+      Use these operations to access Storefront websites.
+      APIs intended to be used directly from an untrusted browser.
```
Comment (noon-dawg) [inline openapi/openapi.yaml:567]:
```suggestion
      Use these operations to access storefront websites.
      
      Storefront operations interact directly with the customer, 
      and provide the customer with access to their own data. 
      
      To use storefront operations, 
      the customer must be issued a JWT with limited permissions. 
      For more information, see [Customer authentication](https://www.rebilly.com/catalog/all/Storefront-account/StorefrontPostLogin) and [Exchange an authentication token](https://www.rebilly.com/catalog/all/Customer-authentication/PostAuthenticationTokenExchange).
      Storefront operations are intended to be used directly from an untrusted browser.
```

**Comment by ashkarpetin** [comment]
js please review

**Review by ninjasimon — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**